.coveragerc

```diff
@@ -0,0 +1,25 @@
+[run]
+source = tuned_lens,tests
+omit = */__init__.py
+
+[report]
+include = tuned_lens/*, tests/*
+# Regexes for lines to exclude from consideration
+exclude_lines =
+    # Have to re-enable the standard pragma
+    pragma: no cover
+
+    # Don't complain about missing debug-only code:
+    def __repr__
+    if self\.debug
+
+    # Don't complain if tests don't hit defensive assertion code:
+    raise AssertionError
+    raise NotImplementedError
+
+    # Don't complain if non-runnable code isn't run:
+    if 0:
+    if __name__ == .__main__.:
+
+    # Don't complain about abstract methods, they aren't run:
```

Comment:
Not necessarily true, abstract methods can have concrete implementations and get called by `super()`. Maybe exclude `pass` and `...` instead that are used as placeholders for abstract methods?